homeassistant/components/plex/services.py

```diff
@@ -28,7 +29,7 @@
 _LOGGER = logging.getLogger(__package__)
 
 
-async def async_setup_services(hass):
+async def async_setup_services(hass: HomeAssistant) -> None:
```

Comment:
The return value was always `True`, and never checked.
I think it is cleaner to drop the return value.